NoteAId-FE/app/screens/AccountScreen.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
```

Comment:
We can also get rid of the screens that are not being used in the main navigation including `ListingDetailsScreen`, `ListingsScreen`, `LoginScreen`, `RegisterScreen`, ...

Reply:
Removed now!